partiql-ast/src/ast.rs

```diff
@@ -179,61 +157,20 @@ pub enum ConflictAction {
     DoNothing,
 }
 
-#[derive(Clone, Debug, PartialEq)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct Expr {
```

Comment:
With these ast changes, `partiql-cli` may also need updated at some point in the future (in particular [ast_to_dot.rs](https://github.com/partiql/partiql-lang-rust/blob/e2cbf2a637ee838040ec41dcd8918f585c00a938/partiql-cli/src/visualize/ast_to_dot.rs)).

Reply:
will fix